table/block_based/partitioned_index_reader.cc

```diff
@@ -82,6 +82,8 @@ InternalIteratorBase<IndexValue>* PartitionIndexReader::NewIterator(
     ro.io_timeout = read_options.io_timeout;
     ro.adaptive_readahead = read_options.adaptive_readahead;
     ro.async_io = read_options.async_io;
+    ro.readahead_size = read_options.readahead_size;
```

Comment:
I enabled readahead_size for index blocks also.

Reply:
I don't know if this is desirable. The index size is much smaller than data, and enabling explicit readahead_size would significantly increase the memory footprint of the iterator, especially if readahead_size is a large value.

Reply:
Ok. I will remove it then.